plugin/evm/vm_test.go

```diff
@@ -124,6 +118,69 @@ func init() {
 	}
 }
 
+func newPrefundedGenesis(
+	balance int,
+	addresses ...common.Address,
+) *core.Genesis {
+	alloc := core.GenesisAlloc{}
+	for _, address := range addresses {
+		alloc[address] = core.GenesisAccount{
+			Balance: big.NewInt(int64(balance)),
+		}
+	}
+
+	genesis := newGenesis()
+	genesis.Alloc = alloc
+
+	return genesis
+}
+
+func newGenesis() *core.Genesis {
+	return &core.Genesis{
+		Config: &params.ChainConfig{
+			AvalancheContext: params.AvalancheContext{
+				BlockchainID: common.Hash{},
+			},
+			ChainID:                         big.NewInt(43111),
+			HomesteadBlock:                  big.NewInt(0),
+			DAOForkBlock:                    big.NewInt(0),
+			DAOForkSupport:                  true,
+			EIP150Block:                     big.NewInt(0),
+			EIP155Block:                     big.NewInt(0),
+			EIP158Block:                     big.NewInt(0),
+			ByzantiumBlock:                  big.NewInt(0),
+			ConstantinopleBlock:             big.NewInt(0),
+			PetersburgBlock:                 big.NewInt(0),
+			IstanbulBlock:                   big.NewInt(0),
+			MuirGlacierBlock:                big.NewInt(0),
+			ApricotPhase1BlockTimestamp:     new(uint64),
+			ApricotPhase2BlockTimestamp:     new(uint64),
+			ApricotPhase3BlockTimestamp:     new(uint64),
+			ApricotPhase4BlockTimestamp:     new(uint64),
+			ApricotPhase5BlockTimestamp:     new(uint64),
+			ApricotPhasePre6BlockTimestamp:  new(uint64),
+			ApricotPhase6BlockTimestamp:     new(uint64),
+			ApricotPhasePost6BlockTimestamp: new(uint64),
+			BanffBlockTimestamp:             new(uint64),
+			CortinaBlockTimestamp:           new(uint64),
+			DUpgradeBlockTimestamp:          new(uint64),
+			CancunTime:                      new(uint64),
+		},
```

Comment:
Hmm I didn't know this package existed, I think that makes sense to use. I think it would still be a good idea to have a function that creates these configs per test as opposed to having a global variable in case tests want to override a default without affecting the state of that config for other tests.